corehq/apps/receiverwrapper/tests/test_submit_errors.py

```diff
@@ -52,6 +56,9 @@ def testSubmitDuplicate(self):
 
         file, res = self._submit('simple_form.xml')
         self.assertEqual(201, res.status_code)
+
+        _, res_openrosa3 = self._submit('simple_form.xml', open_rosa_header=OPENROSA_VERSION_3)
+        self.assertEqual(422, res_openrosa3.status_code)
```

Comment:
@snopoke Yea sounds to me like we definitely don't want to send a 422 in that case, since there is no reason that we would want mobile to try re-submitting, and really no meaningful error for us to worry about from a debugging standpoint. I think it makes sense to continue returning a success response in that case, since for all intents and purposes the form _did_ submit successfully